Automattic/jetpack
# Videopress: Migrate xmlrpc methods to package

#### Changes proposed in this Pull Request:

* Migrate Videopress XMLRPC class into the videopress package
* Consume the package in the Jetpack plugin

#### Other information:


#### Jetpack product discussion


pe4Cmx-40-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Nothing should change, check for regressions
* Upload a video to videopress and make sure the attachment post is created and the metadata is there

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202561463041774
